Test/CoreSDK.Test/Net40/TaskExTests.cs

```diff
@@ -0,0 +1,173 @@
+namespace Microsoft.ApplicationInsights
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Assert = Xunit.Assert;
+
+    [TestClass]
```

Comment:
You could run build in release. Stylecop is automatically checked.

---
In reply to: [73396854](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/267#discussion_r73396854) [](ancestors = 73396854)